examples/OperatorExample.swift

```diff
@@ -116,7 +115,7 @@ public class OperatorExampleViewController: UIViewController {
       return DragSource(subscribedTo: pan, observer: observer).disconnect
     }
 
-    let motionStream = MotionObservable<Int> { observer in
+    _ = MotionObservable<Int> { observer in
```

Comment:
These look like stylistic changes.  Do you need to touch this file to perform the upgrade?